source/topologies/rs-to-sharded.txt

```diff
@@ -0,0 +1,109 @@
+.. _c2c-rs-to-sharded:
+
+=======================================
+Sync a Replica Set to a Sharded Cluster
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+This page outlines the procedure to synchronize data from a replica set to a 
+sharded cluster using :program:`mongosync`. 
+
+``mongosync`` doesn't support sync from a sharded cluster to a replica set.
```

Comment:
[suggestion] I would suggest putting this line in a note!

Reply:
Adding note now!